src/aws_sigv4_internal.erl

```diff
@@ -204,7 +208,7 @@ build_canonical_headers(Signer) ->
         fun({Header, Values}) ->
           [ Header
           , ":"
-          , lists:join(",", lists:map(fun aws_sigv4_utils:trimspace/1, lists:reverse(Values)))
+          , lists:join(",", lists:map(fun string:trim/1, lists:reverse(Values)))
```

Comment:
`string:trim/1` nicely enough does not work on binaries so I think the original needs to remain here unless they're lists and I'm overlooking something.

Reply:
I was wrong 👍 Cool then 👍